Modules/Registration/Common/test/itkMatchCardinalityImageToImageMetricTest.cxx

```diff
@@ -106,5 +106,13 @@ itkMatchCardinalityImageToImageMetricTest(int argc, char * argv[])
     }
   }
 
+
+  typename MetricType::ParametersType parameters = transform->GetParameters();
```

Comment:
Thanks Jon. Note that the `typename` keyword is not necessary here. Line 54 uses MetricType::ParametersType without the keyword. `typename` is only necessary for "dependent types" inside templates.

Reply:
>  if I am not mistaken, in the past, OSX has complained about this

If so, then OSX must have been mistaken! A `typename` like this should never be necessary outside of templates.